pkgs/development/interpreters/ruby/0001-add-post-extract-hook.patch

```diff
@@ -19,3 +28,6 @@ index a88d393..8612901 100644
    end
```

Comment:
I would do `system(ENV['NIX_POST_EXTRACT_FILES_HOOK'], gem_dir)` instead to avoid problems of shell interpolation